ddtrace/internal/ci_visibility/writer.py

```diff
@@ -84,7 +83,7 @@ def __init__(
         sampler=None,  # type: Optional[BaseSampler]
         processing_interval=None,  # type: Optional[float]
         timeout=None,  # type: Optional[float]
-        dogstatsd=None,  # type: Optional[DogStatsd]
+        dogstatsd: Optional[DogStatsd] = None,
```

Comment:
I don't love this getting added in as a single py3 type hint when we might still backport fixes to the event writer to `1.20`.